templates/statsd/statsd-deployment.yaml

```diff
@@ -2,7 +2,7 @@
 ## Airflow StatsD Deployment
 #################################
 kind: Deployment
-apiVersion: apps/v1beta2
+apiVersion: apps/v1
```

Comment:
this is the only change required for compatibility in 1.16 and 1.17. still compatible back to 1.14